packages/welldone-wallet/src/lib/welldone.ts

```diff
@@ -355,6 +378,7 @@ export function setupWelldoneWallet({
           "https://chrome.google.com/webstore/detail/welldone-wallet/bmkakpenjmcpfhhjadflneinmhboecjf",
         deprecated,
         available: installed,
+        useUrlAccountImport: true,
```

Comment:
Let's remove `useUrlAccountImport` on this PR if it is not ready

Reply:
my bad, we actually need this to be true in order to use `buildImportAccountsUrl` for injected wallet 😅